src/appleseed/foundation/platform/python.h

```diff
@@ -38,10 +38,23 @@
     #undef copysign
 #endif
 
+// wrap_python.hpp undefines _DEBUG and then redefines it, losing its value of 1
+// In OSL 1.11.8 osl/platform.h header assumes that _DEBUG=1, otherwise the
+// header can't be compiled
+#ifdef _DEBUG
+  #define _OLD_DEBUG _DEBUG
+#endif
```

Comment:
wrap_python undefines _DEBUG and then redefines it, which makes it lose its value of 1. This value is used in osl/platform.h here
https://github.com/imageworks/OpenShadingLanguage/blob/8ff28a0e062c714b74244cc89fd55461421c5a97/src/include/OSL/platform.h#L229-L232
where OSL_DEBUG = _DEBUG as of here
https://github.com/imageworks/OpenShadingLanguage/blob/8ff28a0e062c714b74244cc89fd55461421c5a97/src/include/OSL/platform.h#L220-L222